visualizer/style/semanticsEditor.css

```diff
@@ -209,3 +217,32 @@ headerBlock real {
   color: #999;
 }
 
+/* `Force Evaluation` entry in `parseTreeMenu` */
+.contextMenu label span {
```

Comment:
This is too generic -- other labels may want to include a span, and they shouldn't automatically get a `margin-left`. Maybe add an appropriately-named class to the span.